src/main/java/org/minions/devfund/mauricioz/Actor.java

```diff
@@ -0,0 +1,51 @@
+package org.minions.devfund.mauricioz;
+
+import java.util.ArrayList;
+
+/**
+ * Actor Class.
+ */
+public class Actor {
+    private String name;
+    private ArrayList<Movie> movies;
+
+    /**
+     * Constructor.
+     */
+    public Actor() {
+        this.name = "";
+        this.movies = new ArrayList<Movie>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)